easybuild/easyconfigs/q/Qt5/Qt5-5.7.0-foss-2016b.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'EB_Qt'
+
+name = 'Qt5'
+version = '5.7.0'
+
+homepage = 'http://qt.io/'
+description = "Qt is a comprehensive cross-platform C++ application framework."
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = [
+    'http://download.qt.io/official_releases/qt/%(version_major_minor)s/%(version)s/single/',
+    'http://download.qt.io/archive/qt/%(version_major_minor)s/%(version)s/single/'
+]
+sources = ['qt-everywhere-opensource-src-%(version)s.tar.xz']
+
+patches = ['%(name)s-%(version)s-workaround-icpc-bug.patch']
```

Comment:
@jrbosch You can drop this, it's only required when building with the Intel compilers